QUARKUS-1027.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1027 OpenShift Service Binding - Tech preview
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1027
+
+The goal of this feature is to provide technology preview support of OpenShift Service Binding for RHOSAK in Quarkus 
+applications.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* None. The testing is executed manually for the Dragonball release by deploying Quarkus Kafka Quickstart in OpenShift 
+  dedicated and binding it to a running RHOSAK instance.
+
+### Impact on resources
+* Currently, a developer preview instance of RHOSAK is used along with OpenShift dedicated running in RHBQ QE AWS quota
+  during the manual testing.
+
+## Future considerations
+* Automate the orchestration of OSD and RHOSAK and implement a test in Quarkus test suite deploying a demo application
```

Comment:
When will be performed / is there any QUARKUS RFE JIRA that will cover this followup automation tasks ?

Reply:
There's no Jira for this, it's simply something we deprioritised because the issue has been lowered to tech preview and had blockers for 2.2.3 release and other priority stuff came up before 2.2.5

Reply:
there is https://issues.redhat.com/browse/QUARKUS-1414, this could be used for automation effort